### DISCUSSION TITLE
how to select content inside current html element?

### DISCUSSION THREAD

**Discussion by noor-tg** [helix-editor/helix#11590] [Q&A]
I want to select content inside element to manipulate it. in view mode or normal mode. but I can't do with helix match key bindings

**Comment by David-Else** [comment]
Please give an exact example of what you are trying to select.

EDIT you probably want this: https://github.com/helix-editor/helix/pull/11158

**Reply by noor-tg** [reply]
Mix match inside element (content only)
Max match tag with content

In the pull request mix is not correct.

**Reply by David-Else** [reply]
Plz check the PR thread, there is some chat about the implementation you can join in with....

**Comment by CptPotato** [comment]
Until something like #11158 is implemented, you should be able to use <kbd>Alt</kbd> <kbd>o</kbd> to grow the selection.
